### ISSUE TITLE
This repo copies code from other author and wrong license

### ISSUE THREAD

**Issue by aggsol** [odwyersoftware/azure-ad-verify-token#11]
tl:dr;

This repo copies code from https://github.com/RobertoPrevato/PythonJWTDemo not naming the original author and removing the original MIT license.

Original author is [@robert](https://github.com/RobertoPrevato) his code is 2 years older than this repo. He explains the code in his blog post: https://robertoprevato.github.io/Validating-JWT-Bearer-tokens-from-Azure-AD-in-Python/

The file https://github.com/odwyersoftware/azure-ad-verify-token/blob/master/azure_ad_verify_token/crypto.py is a carbon copy of https://github.com/RobertoPrevato/PythonJWTDemo/blob/master/jwksutils.py

Expected behaviour: 

a) Restore original author and license (that allows commerical usage)

b) Delete code, repo and package

**Comment by richardARPANET** [comment]
No, it doesn't.
